scripts/scr_company_view/scr_company_view.gml

```diff
@@ -52,18 +52,23 @@ function scr_company_view(company) {
 	sel_uni[1]="Command";
 
 	// This sets up the mans, but not the vehicles
-	var company_length = array_length(obj_ini.name[company]);
 	// Company_lenght is 501, so we check with 500
-	for (var v = 1; v < (company_length-1); v++){
+	var company_length = array_length(obj_ini.TTRPG[company]);
+	for (var v = 0; v < company_length; v++){
```

Comment:
shouldn't this start with `v = 1`?

Reply:
ok, but some lines below, for example, line 76 (right side) you have `ma_race[v]=obj_ini.race[company,v];` on both left and right. Are you sure that changing the starting `v` index won't break those assigments?

Reply:
ok, thanks for the clarification